### PR TITLE
fix windows tests

### PR DESCRIPTION
**Pull request by patricklx**

### CODE REVIEW

File: test-packages/ts-ember-app/tsconfig.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "baseUrl": ".",
     "module": "es6",
     "experimentalDecorators": true,
-    "isolatedModules": true,
+    "isolatedModules": false,
```
Comment (NullVoxPopuli) [inline test-packages/ts-ember-app/tsconfig.json:27]: Why did this need to change?

Is this fallout from using yarn?

Comment (NullVoxPopuli) [reply test-packages/ts-ember-app/tsconfig.json:27]: Oh interesting, looking at the previous types errors, thin fixes an issue with the node types. 
Can you add a note / comment to check if isolatedModules can be flipped back to true when the dep updates?

**Review by NullVoxPopuli — APPROVED** [review]
Approved, but would like a comment added to tsconfig.json

**Comment by NullVoxPopuli** [comment]
yeah, it's technically "jsonc", but with the json extension for some reason

File: test-packages/ts-ember-app/tsconfig.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "baseUrl": ".",
     "module": "es6",
     "experimentalDecorators": true,
-    "isolatedModules": true,
```
[the remark this answers is not in the document]
Comment (NullVoxPopuli) [reply test-packages/ts-ember-app/tsconfig.json:27]: I can only answer the `isolatedModules`, but previous commit's CI: https://github.com/typed-ember/glint/actions/runs/7395334860/job/20118301092#step:8:248 said that @types/node is not compatible with `isolatedModules` -- `@types/node` needs to migrate to using `import type`

[the remark this answers is not in the document]
Comment (patricklx) [reply test-packages/ts-ember-app/tsconfig.json:27]: there are also reports here: https://github.com/microsoft/TypeScript/issues/56913

**Comment by wagenet** [comment]
@patricklx Thanks!

**Pull request merged**